app/components/home/HomeAppThemeSectionComponent.ios.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { View } from 'react-native';
+import { FlatList } from 'react-native-gesture-handler';
+import { useDispatch, useSelector } from 'react-redux';
+
+import ChangeThemeInfoModalComponent from '../appThemes/ChangeThemeInfoModalComponent';
+import Theme from '../../models/Theme';
+import { setSelectedAppTheme } from '../../features/appThemes/appThemeSlice';
+import {largeFontSize, mediumFontSize} from '../../utils/font_size_util';
+import { getStyleOfDevice } from '../../utils/responsive_util';
+
+import HomeAppThemeSampleComponent from './HomeAppThemeSampleComponent';
+import BoldLabelComponent from '../shared/BoldLabelComponent';
+
+const HomeAppThemeSectionComponent = () => {
+  const themes = useSelector(state => state.appTheme.items);
+  const [selectedTheme, setSelectedTheme] = useState(Theme.getDefault());
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const dispatch = useDispatch();
+
+  const listItem = (theme) => {
+    return (
+      <View style={{marginRight: 16}}>
+        <HomeAppThemeSampleComponent theme={theme}
+          onSelect={() => {
+            setSelectedTheme(theme);
+            setIsModalVisible(true);
+          }}
+        />
+      </View>
+    )
+  }
+
+  if (themes.length == 0)
+    return <View/>
+
+  return (
+    <View>
+      <View style={{borderWidth: getStyleOfDevice(0.38, 0.2), marginTop: 26, marginBottom: getStyleOfDevice(18, 14), opacity: 0.4, borderColor: 'white'}}/>
+      <BoldLabelComponent label="ជម្រើសផ្ទៃអេក្រង់"
```

Comment:
@limkimsan, kindly introduce a constant variable(a kind of dictionary) so that in case we need to translate to another language, we can just pass the dictionary wording list to the translator.